source/documentation/using_ci/jenkins.md

```diff
@@ -7,6 +7,10 @@ There are two main approaches to pushing applications to GOV.UK PaaS with [Jenki
 
 Both approaches require you to set up the credentials plugin first.
 
+### Set up dedicated users for testing
```

Comment:
I think we should omit "for testing". Most people will deploy from Jenkins "for real" after running tests outside the PaaS.